src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Converters/ArrowLocationConverter.cs

```diff
@@ -0,0 +1,36 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ArrowLocationConverter.xaml.cs" company="WildGums">
+//   Copyright (c) 2008 - 2018 WildGums. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Orc.Controls
+{
+    using System;
+    using System.Windows;
+    using System.Windows.Controls;
+    using Catel.MVVM.Converters;
+
+    public class ArrowLocationConverter : ValueConverterBase
+    {
+        protected override object Convert(object value, Type targetType, object parameter)
+        {
+            if (targetType == typeof(Dock))
+            {
+                return Enum.Parse(targetType, value.ToString(), true);
```

Comment:
value can be null. Try to use ObjectTostringConverter or Helper and check for null.